run.py

```diff
@@ -61,14 +61,24 @@ def run_threads(sim: bool) -> None:
                 logging.error("Flight process terminated, restarting")
                 flight_process: Process = init_flight(flight_args)
                 flight_process.start()
-    except KeyboardInterrupt:
+    except KeyboardInterrupt or AttributeError:
```

Comment:
I'm not sure if this is the right syntax for catching multiple errors. See https://stackoverflow.com/questions/6470428/catch-multiple-exceptions-in-one-line-except-block

Let me know if I'm wrong about this.